src/quacc/recipes/vasp/core.py

```diff
@@ -320,3 +325,61 @@ def non_scf_job(
         additional_fields={"name": "VASP Non-SCF"},
         copy_files={prev_dir: ["CHGCAR*", "WAVECAR*"]},
     )
+
+
+@job
+def freq_job(
+    atoms: Atoms,
+    preset: str | None = "BulkSet",
+    energy: float = 0.0,
+    temperature: float = 298.15,
+    pressure: float = 1.0,
+    vib_kwargs: VibKwargs | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+    **calc_kwargs,
+) -> VibThermoSchema:
+    """
+    Run a frequency job and calculate thermochemistry.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    preset
+        Preset to use from `quacc.calculators.vasp.presets`.
+    energy
+        Potential energy in eV. If 0, then the output is just the correction.
+    temperature
+        Temperature in Kelvins.
+    pressure
+        Pressure in bar.
+    vib_kwargs
+        Dictionary of kwargs for the [ase.vibrations.Vibrations][] class.
+    copy_files
+        Files to copy (and decompress) from source to the runtime directory.
+    **calc_kwargs
+        Custom kwargs for the Vasp calculator. Set a value to
+        `None` to remove a pre-existing key entirely. For a list of available
+        keys, refer to [quacc.calculators.vasp.vasp.Vasp][].
+
+    Returns
+    -------
+    VibThermoSchema
+        Dictionary of results, specified in [quacc.schemas.ase.summarize_vib_and_thermo][].
+        See the type-hint for the data structure.
+    """
+    calc_defaults = {"ediff": 1e-8, "isym": 0, "lcharg": False, "lwave": True, "nsw": 0}
```

Comment:
Would you be okay with `ediff = 1e-7`? This is what Materials Project uses for accurate forces, so it'd be quite consistent. But if you disagree, that's fine. 

Also, once the Custodian patch is merged, we can/should add `sigma=-5` back in so we can ensure the forces are very accurate.